### PR TITLE
Add OpenVisualTraceroute.app v1.6.5

### PR DESCRIPTION
**Pull request by efeldhusen**
After making all changes to the cask:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [X] `brew cask style --fix {{cask_file}}` reports no offenses.
- [X] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [X] Named the cask according to the [token reference].
- [X] `brew cask install {{cask_file}}` worked successfully.
- [X] `brew cask uninstall {{cask_file}}` worked successfully.
- [X] Checked there are no [open pull requests] for the same cask.
- [X] Checked the cask was not already refused in [closed issues].
- [X] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by efeldhusen** [comment]
When I tried to format the URL, as suggested in the guide, I could not make it work. 

https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#sourceforgeosdn-urls

File: Casks/openvisualtraceroute.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  # downloads.sourceforge.net/openvisualtrace was verified as official when first introduced to the cask
+  url "https://downloads.sourceforge.net/openvisualtrace/#{version}/OpenVisualTraceRoute#{version}.dmg"
+  appcast 'https://sourceforge.net/projects/openvisualtrace/rss',
+          checkpoint: 'a9d378d7cbe829d31479a38307d89b95509520e2ccd06ca2b3b49c7db69f498a'
+  name 'OpenVisualTraceroute'
+  homepage 'http://visualtraceroute.net/'
+
+  app 'OpenVisualTraceroute.app'
+
+  zap delete: '~/ovtr'
```
Comment (commitay) [inline Casks/openvisualtraceroute.rb]: `zap trash:` 

Should it be `~/.ovtr`?

File: Casks/openvisualtraceroute.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  caveats do
+    depends_on_java('6')
```
Comment (commitay) [inline Casks/openvisualtraceroute.rb]: Is it Java 6 only?

**Review by commitay — CHANGES_REQUESTED** [review]

**Review by commitay — APPROVED** [review]

**Comment by miccal** [comment]
Thank you for your contribution.

**Comment by efeldhusen** [comment]
Thank you!

**Pull request merged**